src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -64,13 +74,47 @@ public override bool Equals(System.Object obj)
               eventTime == sec.eventTime;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static bool operator ==(GestureInfoType arg1, GestureInfoType arg2)
+        {
+            return arg1.Equals(arg2);
+        }
+
+        /// <summary>
+        /// The != operator.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static bool operator !=(GestureInfoType arg1, GestureInfoType arg2)
+        {
+            return !arg1.Equals(arg2);
+        }
+
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override int GetHashCode()
         {
             return Tuple.Create((int)type, xBeg, xEnd, yBeg, yEnd, (int)state, eventTime).GetHashCode();
         }
     }
 
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class GestureInfoEventArgs : EventArgs
+    {
+        public GestureInfoType gestureInfo { get; internal set; }
```

Comment:
Please make property name to start with capital alphabet.

Reply:
Okay! Changed to `GestureInfo`.